### PR TITLE
Fix UI gamblingscreen

### LINKED ISSUE #311: Enhance Spin the Wheel

### PR DESCRIPTION
**Pull request by AlexandreBeeDeeBop**
Content of this PR:
The rarities written on the wheel are now correctly centered. The spin button indicates how much it costs to spin. I also accidentally increased the code coverage which was already excellent.

Closes #311

### CODE REVIEW

File: app/src/main/java/com/github/se/cyrcle/ui/gambling/GamblingScreen.kt [hunk 1]
```diff
@@ -209,15 +209,29 @@ fun WheelView(
 
           val startAngle = index * segmentAngle
           val middleAngle = Math.toRadians((startAngle + segmentAngle / 2.0))
-          val textRadius = radius * 0.6f
+          val textRadius = radius * 0.65f // Moved closer to edge (was 0.6f)
```
Comment (andrepeiry) [inline app/src/main/java/com/github/se/cyrcle/ui/gambling/GamblingScreen.kt:212]: No need to add a comment like this for a float value change (even more for UI)

**Review by andrepeiry — APPROVED** [review]
Not a lot to say, be sure to harass your reviewer next time such that this kind of old PR doesn't stay waiting on them.

**Review by LouisJamesVasseur — APPROVED** [review]
Nothing to add here, good work. Sorry this PR has stayed here so long w/o review (we'd discussed that it still needed a couple changes and I was never re-requested afterwards so didn't take too deep a look)

**Comment by AlexandreBeeDeeBop** [comment]
quick rebase onto main, ready to merge, nothing has changed since then

**Pull request merged**